source/read/specify-documents-to-return.txt

```diff
@@ -0,0 +1,200 @@
+operation by using the following methods:
+
+- :ref:`limit() <cpp-return-documents-limit>`: Specifies the maximum number of documents
+  to return from a query.
+- :ref:`sort() <cpp-return-documents-sort>`: Specifies the sort order for the returned documents.
+- :ref:`skip() <cpp-return-documents-skip>`: Specifies the number of documents to skip before
+  returning query results.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your C++ application, instantiate a ``client`` that connects to an Atlas cluster
+and assign the following values to your ``db`` and ``collection`` variables:
+
+.. literalinclude:: /includes/read/limit-skip-sort.cpp
+    :language: cpp
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets, see
+the :ref:`<cpp-get-started>` tutorial.
+
+.. _cpp-return-documents-limit:
+
+Limit
+-----
+
+To specify the maximum number of documents returned from a read operation, create
+an instance of the ``mongocxx::options::find`` class and set its ``limit`` field.
+Then, pass your ``mongocxx::options::find`` instance as a parameter to the
+``find()`` method.
+
+The following example finds all restaurants that have a ``cuisine`` field value
+of ``"Italian"`` and limits the results to ``5`` documents:
+
+.. io-code-block::
+
+   .. input:: /includes/read/limit-skip-sort.cpp
+      :start-after: start-limit-method
+      :end-before: end-limit-method
+      :language: cpp
+      :dedent:
+
+   .. output::
+
+      { "_id" : { "$oid" : "..." }, ..., "name" : "Philadelphia Grille Express", "restaurant_id" : "40364305" }
+      { "_id" : { "$oid" : "..." }, ..., "name" : "Isle Of Capri Restaurant", "restaurant_id" : "40364373" }
+      { "_id" : { "$oid" : "..." }, ..., "name" : "Marchis Restaurant", "restaurant_id" : "40364668" }
+      { "_id" : { "$oid" : "..." }, ..., "name" : "Crystal Room", "restaurant_id" : "40365013" }
+      { "_id" : { "$oid" : "..." }, ..., "name" : "Forlinis Restaurant", "restaurant_id" : "40365098" }
+
+.. tip::
+
+   The preceding example returns the first five documents matched by the query
+   according to their :manual:`natural order </reference/glossary/#std-term-natural-order>`
+   in the database. The following section describes how to return the documents
+   in a specified sort order.
+
+.. _cpp-return-documents-sort:
+
+Sort
+----
+
+To return documents in a specified order, create an instance of the ``mongocxx::options::find``
+class and set its ``sort`` field. Then, pass your ``mongocxx::options::find`` instance as a
+parameter to the ``find()`` method.
+
+
+To set the ``sort`` field, call the ``sort()`` setter method on your ``mongocxx::options::find``
```

Comment:
Based on the API docs, seems like the find() method and limit() are the same type of method, so adding setter might not clarify/help this sentence.

Reply:
sort() and limit() are setter methods, since they set fields of a `mongocxx::options::find` object; find() is not. But I can remove the word "setter" here if it's confusing